manifests/v/VirtusLab/ScalaCLI/v1.0.2/VirtusLab.ScalaCLI.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created with WinGet Releaser v2 using Komac v1.8.2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: VirtusLab.ScalaCLI
+PackageVersion: v1.0.2

```

```suggestion
PackageVersion: 1.0.2
```
This should match control panel exactly. After this is updated, the folder will need to be renamed